lxc/config/remote.go

```diff
@@ -105,7 +131,7 @@ func (c *Config) GetInstanceServer(name string) (lxd.InstanceServer, error) {
 		return d, nil
 	}
 
-	// HTTPs
+	// HTTPS
```

Comment:
Should be a separate commit really.

Reply:
Won't address this as it has already been merged.